endoreg_db/services/video_import.py

```
@@ -192,7 +191,197 @@ def import_and_anonymize(
+            raise ValueError("Raw file not found or invalid path")
+        
+    if frame_cleaning_available and video_file_obj.raw_file:
+        if not just_anonymization:
+            logger.info(f"Anonymizing VideoFile UUID: {video_file_obj.uuid} with processor {processor_name}")
+            if method=="masking":
+                input_video = video_file_obj.raw_file.path
+                mask_config = mask_config
+                return FrameCleaner._mask_video(input_video, mask_config, video_file_obj.processed_file.path)
+            else:
+                logger.warning(f"Unknown anonymization method: {method}. TBA!.")
+                method = "masking"
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)